src/config/constants/index.js

```diff
@@ -1,5 +1,22 @@
-const DEFAULT_TIME_OUT = {
-  TOAST: 4000,
+export const DEFAULT_VALUES = {
+  TOAST_TIMEOUT: 4000,
+  PROJECT_TITLE_MAX_LENGTH: 15,
 };
 
-export default DEFAULT_TIME_OUT;
+export const MESSAGE = {
+  PROJECT_SUCCESS_CREATE: "생성 되었습니다!",
+  PROJECT_SUCCESS_EDIT: "수정 되었습니다!",
+  PROJECT_SUCCESS_DELETE: "삭제 되었습니다!",
+  PROJECT_FAIL: "문제가 발생했습니다. 다시 시도해주세요.",
+  BAD_REQUEST: "잘못된 접근입니다.",
+};
+
+export const FORM = {
+  CREATE: "create",
+  CREATE_TITLE: "새로운 프로젝트 이름",
+  EDIT: "edit",
+  EDIT_TITLE: "프로젝트 이름 수정",
+};
+
+export const PROJECT_GUIDE_INFO =
+  "      자세한 사용 방법은 우측의 '사용 가이드 문서' 링크를 통해 확인하실 수 있습니다.\n\n아래의 스크립트에 프로젝트 ID와 secret-key를 입력한 후, 스크립트 전체를 복사하여 사용하시는 html의 <body> Tag 내부 최하단에 붙혀 넣어주세요.";
```

Comment:
```suggestion
  "자세한 사용 방법은 우측의 '사용 가이드 문서' 링크를 통해 확인하실 수 있습니다.\n\n아래의 스크립트에 프로젝트 ID와 secret-key를 입력한 후, 스크립트 전체를 복사하여 사용하시는 html의 <body> Tag 내부 최하단에 붙혀 넣어주세요.";
```
혹시 일부러 띄어쓰기 넣으신 건가요?!

Reply:
네! 해당부분에 텍스트 앞에 종 아이콘이 와야해서 띄어쓰기를 했는데, 생각해보니 css로 해결하는게 더 좋을 것 같습니다! 수정해보겠습니다!